### DISCUSSION TITLE
mails r holded in queue

### DISCUSSION THREAD

**Discussion by reetikpant** [postalserver/postal#2069] [Help with using Postal]
all my mails are going to queue, i have stopped postal and restarted it, still all new mails im sending are going to queue are either failed or pending. what to do?

**Comment by reetikpant** [comment]
![pop](https://user-images.githubusercontent.com/97339805/178656252-6205ced2-632e-4136-b08c-6c552d9c3434.png)

here's screenshot of postal status @willpower232

**Reply by reetikpant** [reply]
also this is what i'm getting for gmail, same for hotmail @willpower232 

No SMTP servers were available for gmail.com. Tried gmail-smtp-in.l.google.com, alt1.gmail-smtp-in.l.google.com, alt2.gmail-smtp-in.l.google.com, alt3.gmail-smtp-in.l.google.com, and alt4.gmail-smtp-in.l.google.com

**Reply by willpower232** [reply]
It sounds like your server provider is blocking connection to those servers (or they're both blocking you) so if you can't resolve this with your server provider then you might be out of luck with them

**Reply by reetikpant** [reply]
server as in? contabo?

**Reply by reetikpant** [reply]
im using contabo vps, so in this case server is contabo or something else?

**Reply by reetikpant** [reply]
also @willpower232 can i delete my queue? if yes then how? it'll help ?
